src/server/cli.ts

```diff
@@ -30,19 +30,13 @@ import {
 } from "./cli/options";
 import runTurnilo from "./cli/run-turnilo";
 import { parseCredentials } from "./cli/utils";
-import { readVersion } from "./version";
 
-let version: string;
-try {
-  version = readVersion();
-} catch (e) {
-  program.error(`Failed to read turnilo version. Error: ${e.message}`);
-}
+declare const VERSION: string;
```

Comment:
what do you think about adding a comment about where is it actually defined?

Reply:
changed variable name to be self explanatory